test/Storage.C

```diff
@@ -164,6 +174,8 @@ TEST(Storage, CSeriesAPI) {
       st.x = i;
       st.y = 3.14159 * static_cast<double>(i);
       st.z = makeString("string_" + str::from(i));
+      st.b = true;
```

Comment:
let's test bools and fixed-size arrays?
`
st.b = (i % 2) == 0 
`

Reply:
`
EXPECT_TRUE(c.compileFn<bool()>("[b|{b=b}<-cf.series_test] == take(1000,cycle([true, false]))")());
`
You would need to define `cycle` if it doesn't exist yet

Reply:
I have update the test case.  

st.z = makeString("string_" + str::from(i));
 st.b = i % 2 == 0;
 memcpy(st.v, st.z->data, 8);

 EXPECT_TRUE(c.compileFn<bool()>("[b|{b=b}<-cf.series_test][:0] == [x%2==0|{x=x}<-cf.series_test][:0]")());
 EXPECT_TRUE(c.compileFn<bool()>("[v|{v=v}<-cf.series_test][:0] == [z[0:8]|{z=z}<-cf.series_test][:0]")());

Reply:
nice, thanks!